kv.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the LGPL, see LICENCE file for details.
+
+package simplekv
+
+import (
+	"context"
+	"time"
+
+	errgo "gopkg.in/errgo.v1"
+)
+
+var (
+	// ErrNotFound is the error cause used when an identity cannot be
+	// found in storage.
+	ErrNotFound = errgo.New("not found")
+
+	// ErrDuplicateKey is the error cause used when trying to set a
```

Comment:
This comment seems out of date with the usage now.

Reply:
fixed